packages/documentation/src/stories/foundation/elevation/elevation.styles.scss

```diff
@@ -0,0 +1,31 @@
+@use '@swisspost/design-system-styles/core.scss' as post;
+
+.elevation-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: post.$size-big;
+}
+
+.elevation-tile {
+  width: 100%;
```

Comment:
```suggestion
```

This causes the tile element to be 100% width, which is not necessary.
If we remove this CSS property the elevation-tiles can occur side by side and will wrap, if there is to little space:

![image](https://user-images.githubusercontent.com/9716662/235852423-c31cd6d2-cb5d-454b-9441-2d258c8a22ce.png)